trustee-on-azure/01-infrastructure-setup-Azure/01-creating-custom-rhcos-image.md

```diff
@@ -0,0 +1,98 @@
+# Create Customized RHCOS image with 'trustee-attester' + 'aa-client-service' in Initrd
+
+**Authors:** demeng@redhat.com, hhei@redhat.com \
+**Jira Issue:** [VIRTWINKVM-1188](https://issues.redhat.com/browse/VIRTWINKVM-1188)  
+**References:**
+- [Custom CoreOS Disk Images](https://github.com/coreos/custom-coreos-disk-images)
+- [Quay.io Repository](https://quay.io/repository/)
+- [Latest RHCOS Images](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/)
+
+(Explanatory statement: Include trustee-attester binary file + .sh/.service files in the initrd to ensure the passphrase from the KBS-server can be fetched at the initramfs stage.).
+
+
+### Step 1: Get authentication credentials
+```bashV
+Create ~/.config/containers/auth.json by clicking https://console.redhat.com/openshift/create/local, then copy or download secret.
+```
+
+### Step 2. Deploy Fedora 42 environment
+```bash
+sudo dnf update -y
+sudo setenforce 0
+sudo sed -i -e 's/SELINUX=enforcing/SELINUX=permissive/' /etc/selinux/config
+sudo dnf install -y osbuild osbuild-tools \
+         osbuild-ostree podman jq xfsprogs \
+         e2fsprogs dosfstools genisoimage \
+         squashfs-tools erofs-utils syslinux-nonlinux \
+mkdir ~/test/create_cos_image; cd ~/test/create_cos_image
+# copy the trustee-attester binary file to this path.
+```
+
+
+### Step 3. Get the relevant code
+This repo [initrd-trustee_plus](https://gitlab.com/6-dehan/initrd-trustee_plus.git) contains several scripts that used to build container images for RHCOS.
+```bash
+The repo is stored in the parent directory.
+```
+
+### Step 4. Pull container image and query the digest
+```bash
+podman pull quay.io/openshift-release-dev/ocp-v4.0-art-dev:419.96.202505021444-0-coreos
```

Comment:
nit: can you please use a variable for the digest, it makes it easier to copy paste the command